x-pack/test/case_api_integration/basic/tests/cases/find_cases.ts

```diff
@@ -6,246 +6,454 @@
  */
 
 import expect from '@kbn/expect';
+import supertestAsPromised from 'supertest-as-promised';
```

Comment:
nit: As of 2.0.0 `supertest` supports native promised and `supertest-as-promised` is not needed.

Reply:
Yeah it's just needed for the type because I'm putting all the promises in an array like this:

```
const responsePromises: supertestAsPromised.Test[] = [];
```

That was the location that vscode thought the type was defined 🤷‍♂️